docs/src/routes/docs/installation.svx

```diff
@@ -6,13 +6,13 @@ name: Installation
   import { Tabs } from 'attractions';
   import CopyableCode from 'src/components/docs/copyable-code.svelte';
 
-  const packageManagers = ['npm', 'Yarn']
+  const packageManagers = ['npm', 'Yarn', 'pnpm']
   let packageManager = packageManagers[0];
 </script>
 
 # Installation
 
```

Comment:
Shouldn't the README be updated as well? Or do we replace it entirely with a link to the docs?

Reply:
Yeah it should